### ISSUE TITLE
Add email template on asking for citations of other datasets

### ISSUE THREAD

**Issue by laijasmine** [NCEAS/datateam-training#220]
- [ ] Integrate part of data processing workflow 
- [ ] follow up with them a couple years down the line again
- [ ] Add some email templates (maybe based on the pre-issued doi template)

**Comment by KellyWang1126** [comment]
@laijasmine Do you mean we should follow up with PI about the citations of other datasets a couple years later after the data package is published?

**Comment by laijasmine** [comment]
Yeah I haven't thought this out completely yet.. I've assigned this to myself so you can ignore this one for now!
